debian/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM debian:stretch
+MAINTAINER Shaun Jackman <sjackman@gmail.com>
+
+# Set apt in non-interactive mode during the image build
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Update debian package list
+RUN apt-get update
+
+# Set up UTF-8
+RUN apt-get install -y apt-utils
+RUN apt-get install -y locales
```

Comment:
I was going to say that if merging these results in an error message, then something is wrong. Which warning do you see now? Please post it here as a regular reply and reference this message (right click on the "X hours ago" next to my or your user name and copy the link).